packages/authorize/src/api.ts

```diff
@@ -0,0 +1,102 @@
+import { avUserPermissionsApi, avRegionsApi } from '@availity/api-axios';
+
+import type { Permission, RequestedPermissions, RequestedResources } from './types';
+
+export const getRegion = async (region?: boolean | string): Promise<string | undefined> => {
+  if (region === true) {
+    const resp = await avRegionsApi.getCurrentRegion();
+
+    return resp?.data?.regions?.[0]?.id;
+  }
+
+  return region || undefined;
+};
+
+export const getPermissions = async (
```

Comment:
This could use [usePermissions](https://github.com/Availity/availity-react/blob/master/packages/hooks/src/usePermissions.js)